src/docfx/build/page/Markup.cs

```diff
@@ -80,6 +88,18 @@ internal static class Markup
             }
         }
 
+        private static string ToHtml(string markdown, MarkdownPipelineType pipelineType)
```

Comment:
make it `MarkdownPipeline GetPipeline(pipelineType)` to be shareable with `Parse`